analytics/core.go

```diff
@@ -88,6 +89,7 @@ type UsersyncInfo struct {
 
 // NotificationEvent object of a transaction at /event
 type NotificationEvent struct {
-	Request *EventRequest   `json:"request"`
-	Account *config.Account `json:"account"`
+	Request         *EventRequest   `json:"request"`
+	Account         *config.Account `json:"account"`
+	ActivityControl privacy.ActivityControl
```

Comment:
Should this be removed now that we have the Runner taking care of all activity enforcement?

Reply:
Yes, sure, removed.